packages/@contentful--app-scripts/lib/activate/activate-bundle.js

```diff
@@ -0,0 +1,43 @@
+const ora = require('ora');
+const chalk = require('chalk');
+const { throwError } = require('../utils');
+const { createClient } = require('contentful-management');
+
+async function activateBundle({ accessToken, organization, definition, bundleId }) {
+  const activationSpinner = ora('Activating your bundle').start();
+  const client = createClient({ accessToken });
+  const currentOrganization = await client.getOrganization(organization.value);
+  const currentDefinition = await currentOrganization.getAppDefinition(definition.value);
+
+  currentDefinition.bundle = {
+    sys: {
+      id: bundleId,
+      linkType: 'AppBundle',
+      type: 'Link',
+    },
+  };
+  currentDefinition.src = undefined;
```

Comment:
Slightly more correct:
```suggestion
  delete currentDefinition.src;
```